meta-resin-common/recipes-support/resin-expand/resin-filesystem-expand/resin-filesystem-expand

```diff
@@ -1,7 +1,5 @@
 #!/bin/bash
 
-set -e
```

Comment:
I would probably consider this change type a patch. And to be honest I'd remove any changelog generation for this specific modification.